platforms/Windows/bld/asserts/bld.asserts.wxs

```diff
@@ -1,3 +1,11 @@
 <Wix xmlns="http://wixtoolset.org/schemas/v4/wxs">
+  <?define ToolchainRoot = $(ImageRoot)\Toolchains\$(ProductVersion)+Asserts?>
+  <?define VariantName = asserts ?>
+  <?define VariantUpgradeCode = $(BldAssertsUpgradeCode)?>
+  <?define VariantProductName = !(loc.BldAsserts_ProductName)?>
+  <?define VariantCabName = bld.asserts.cab?>
+  <?define ToolchainVersionedVariantDirectory= ToolchainVersionedAsserts ?>
+  <?define VariantEnvironmentComponentGUID = ab52b870-23ee-42e8-9581-3fcbdfb9228c?>

```

Comment:
Does this need to be here or should it go into shared.wxs along with the other GUID constants?

Reply:
the other ones need to change between major upgrades. this one just needs to have a guid to identify since the Id (`UserPathVariable`) is not unique enough. it needs to be different from the other variant. so this is why i put it here. 

We have 3 of these, in bld (toolchain\usr\bin), rtl (runtimes\usr\bin) and windowssdk(sdks\windows.sdk). we could pull them all out if you want, but not sure if that will add much value